libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -61,6 +65,25 @@ static DNNDataType precision_to_datatype(precision_e precision)
     }
 }
 
+static DNNReturnType parse_options(OVContext *dst, const char *src_opt, ie_available_devices_t *a_dev)
```

Comment:
should not pass a_dev here, we'll add more options, we cannot add a function parameter for each possible option.

for invalid device, we can rely on the return value of ie_core_load_network.  If it returns error, we can then call ie_core_get_available_devices and list all the supported devices.

Reply:
I think you mean we can verify if the value of device_type match the available device in ff_dnn_load_model_ov() rather than in parse_option() ?
becasue the return of ie_core_load_network can not reflect if the specified device ready or not. (I have checked the IESatatusCode in OpenVINO api just now, no such value like: DEVICE_NOT_FOUND or NO_SUCH_DEVICE)

Reply:
just ignore this since we will remove this function,